src/ResourceManager/Automation/Commands.Automation/Properties/Resources.resx

```diff
@@ -201,6 +201,10 @@
     <value>The variable was not found. Variable name {0}.</value>
     <comment>Automation</comment>
   </data>
+  <data name="InputValueSerializationTimedOut" xml:space="preserve">
+    <value>Input value cannot be serialized to json and the operation had timed-out in 30 secs. Name {0}. Please execute '$jsonString=ConvertTo-Json -InputObject $inputObject -Depth 4' and review $jsonString, $jsonString.Length. Either the input is a complex object or length of the object is bigger.</value>
```

Comment:
May be better to have the timeout value passed in rather than hard-coded in the string.

For the error itself, I would suggest:

```none
Input object '{0}' could not be serialized to json.  Operation had timed out in {1} second(s).   The input object was too complex or resulted in too large of a json.
```

Reply:
Thank you. I will fix this

Reply:
![image](https://user-images.githubusercontent.com/16600949/37737958-aeaafed8-2d12-11e8-952d-dd8b5ed18b62.png)